### DISCUSSION TITLE
when I share a link with someone, is that a snapshot ?

### DISCUSSION THREAD

**Discussion by dhk** [excalidraw/excalidraw#8432] [Q&A]
I have just now discovered excalidraw and I'm loving it. 

I'm curious about the sharing feature - is that a snapshot in time or will the link track changes that happen to the diagram after I share the URL with someone else. 

For example, [there's this](https://excalidraw.com/#json=u2_PToiJpvBuCcbFeVc3F,x0yj8Whv0pHFcQLeH3tbLg), which is what my diagram looks like right now.
![PM+AI](https://github.com/user-attachments/assets/94eabef5-769e-4666-b4af-ca66f4710b74)

**Comment by Mrazator** [comment]
> I'm curious about the sharing feature - is that a snapshot in time or will the link track changes that happen to the diagram after I share the URL with someone else.

If you create a live collaboration session, it will keep being updated. However, the read-only link you've only captured the diagram at the point of link creation. Both are non-expiring ATM, though it might change in the future.
